src/actions/ForgotPassword.js

```diff
@@ -0,0 +1,64 @@
+import { Form, Message } from "semantic-ui-react";
```

Comment:
This is a component, it should be in the components' folder. And create actions for it.